app/models/merchant.rb

```diff
@@ -5,5 +5,7 @@ class Merchant < ApplicationRecord
   has_many :transactions, through: :invoices
   has_many :customers, through: :invoices
 
+  enum status: ["Enabled", "Disabled"]
```

Comment:
can you refactor this to be a hash (i.e. {"Enabled => 0, "Disabled" => 1}) so if we have to update the merchant status there is a easy place to find what number is tied to what status.

Reply:
Fixed that issue